<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DRaceGUI</class>
 <widget class="QMainWindow" name="DRaceGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DRaceGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="4">
     <widget class="QLineEdit" name="drace_path_input"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>DRace - Path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>DynamoRIO - Path</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="5">
     <widget class="QPushButton" name="run_button">
      <property name="text">
       <string>RUN</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="4">
     <widget class="QTextBrowser" name="command_output">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Executable (exe under test)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="drace_link">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://www.github.com/siemens/drace/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;DRace on GitHub!&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextBrowserInteraction</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="4">
     <widget class="QLineEdit" name="exe_input"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Configuration File</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Command</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QCheckBox" name="report_creation">
      <property name="text">
       <string>Report</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="drace_path_btn">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QCheckBox" name="msr_box">
      <property name="text">
       <string>MSR</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QLineEdit" name="dr_path_input">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DRaceGUI</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="dr_debug">
      <property name="text">
       <string>Debug Mode</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QRadioButton" name="fasttrack_btn">
      <property name="text">
       <string>Fasttrack</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="config_browse_btn">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="5">
     <widget class="QLineEdit" name="flag_input"/>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>DRace Flags (use single quotes when needed)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="dr_path_btn">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QPushButton" name="exe_path_btn">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4">
     <widget class="QLineEdit" name="config_file_input"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Select Detector Backend</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QRadioButton" name="tsan_btn">
      <property name="text">
       <string>TSAN</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QRadioButton" name="dummy_btn">
      <property name="text">
       <string>Dummy</string>
      </property>
     </widget>
    </item>
    <item row="16" column="4">
     <widget class="QPushButton" name="copy_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Copy to 
 Clip- 
 board</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDRace">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Config"/>
    <addaction name="actionSave_Config"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionConfigure_Report"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDRace"/>
   <addaction name="menuReport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfigure_Report">
   <property name="text">
    <string>Configure Report</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>dr_path_input</tabstop>
  <tabstop>dr_path_btn</tabstop>
  <tabstop>dr_debug</tabstop>
  <tabstop>drace_path_input</tabstop>
  <tabstop>drace_path_btn</tabstop>
  <tabstop>tsan_btn</tabstop>
  <tabstop>dummy_btn</tabstop>
  <tabstop>fasttrack_btn</tabstop>
  <tabstop>config_file_input</tabstop>
  <tabstop>config_browse_btn</tabstop>
  <tabstop>exe_input</tabstop>
  <tabstop>exe_path_btn</tabstop>
  <tabstop>flag_input</tabstop>
  <tabstop>command_output</tabstop>
  <tabstop>copy_button</tabstop>
  <tabstop>run_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
